### PR TITLE
add @2045's testimonial

### LINKED ISSUE #173: Add @2045's lovely testimonial
You may publish this:

"The foundation's code stewards helped us in a great way to make the step to maintaining our platform code fully open.
We had two sessions with code stewards enabling the team to determine the work to be done, making the step to working fully open.
Since some time the code is maintained at https://gitlab.com/logius/cloud-native-overheid where we work with more then one team on this code base.

A special thanks to Eric Herman and Jan Ainali
Martin Dias d'Ullois - Product Owner, Logius"

_Originally posted by @2045 in https://github.com/publiccodenet/publiccode.net/issues/70#issuecomment-781475244_

### PR DESCRIPTION
**Pull request by ElenaFdR**
Fixes #173.

Work in progress! Still to do:

- [x] finesse testimonial placement
- [x] replace Mirjam placeholder photo with ~~@2045's photo~~ (edit: @2045 prefers to not have his photo online)
- [x] confirm final testimonial wording with @2045

### CODE REVIEW

**Comment by ElenaFdR** [comment]
Re testimonial location:
- it feels like a natural fit for support stewardship, but that's quite far below the fold
- the quotation is long enough that it doesn't play nicely with the default layout (your thoughts, @angelplasma?)

File: codebase-stewardship/index.html [hunk 1]
```diff
@@ -27,13 +28,21 @@ <h1>Everything your project needs to be sustainable, collaborative and open.</h1
...
   <div><h2>Making sure the code, tests and documentation are always up to the standards</h2>
 
   <p>
     We check every merge request on code, policy and documentation quality in our <strong>codebase auditing</strong> process. This enables agile development for trustworthy codebases for public organizations and their collaborators.
   </p>
 
   </div>
+
+  <blockquote>
+    <img src="../photos/mirjam-hierarchy.jpg" alt="Martin Dias d'Ullois" class="testimonial-photo">
```
Comment (bvhme) [inline codebase-stewardship/index.html:41]: Can we do this with a generic illustration or a grey circle? Maybe a photo of the Logius building or something?

File: codebase-stewardship/index.html [hunk 2]
```diff
@@ -64,7 +73,9 @@ <h1>Everything your project needs to be sustainable, collaborative and open.</h1
     To drive <strong>developer success</strong> we help with (end)user support, packaging, and distributing official versions as well as developing training materials and certifications. We also help vendors such as cooperatives, NGOs and for profit companies to sell, implement and support the product.
   </p>
   </div>
-  <img alt="illustration, ecosystem" src="https://publiccodenet.github.io/illustrations/illustrations/eco-system.svg">
+
+<img alt="illustration, ecosystem" src="https://publiccodenet.github.io/illustrations/illustrations/eco-system.svg">
```
Comment (bvhme) [inline codebase-stewardship/index.html:77]: Indentation seems correct

```suggestion
  <img alt="illustration, ecosystem" src="https://publiccodenet.github.io/illustrations/illustrations/eco-system.svg">
```

**Review by bvhme — CHANGES_REQUESTED** [review]
Getting Martin's quote in would help me in my work, I was working on a pull request when I saw this one. Let me know if you'd rather I made one.

File: codebase-stewardship/index.html (hunk 1 above)
Comment (ElenaFdR) [reply codebase-stewardship/index.html:41]: [Logius uses the overheid crest](https://twitter.com/logius_minbzk) as their logo - do you think we can just borrow that?

**Review by angelplasma — APPROVED** [review]
New quote and SVG look good.

**Review by bvhme — APPROVED** [review]
Great! Looks good to me.

**Pull request merged**